tests/functional/Codeception/Docker.php

```diff
@@ -512,7 +554,7 @@ public function runEceToolsCommand(
      */
     public function seeInOutput(string $text)
     {
-        \PHPUnit\Framework\Assert::assertContains($text, $this->output);
+        Assert::assertContains($text, $this->output);
```

Comment:
This method should be moved to a helper so it can be a part of the `CliTester` object. Something for when we refactor these classes in the future.